pkgs/tools/networking/pihole-ftl/default.nix

```diff
@@ -0,0 +1,59 @@
+{ lib
+, cmake
+, fetchFromGitHub
+, gmp
+, libidn
+, nettle
+, readline
+, stdenv
+}:
+
+stdenv.mkDerivation rec {
+  pname = "pihole-ftl";
+  version = "5.8.1";
+
+  src = fetchFromGitHub {
+    owner = "pi-hole";
+    repo = "FTL";
+    rev = "v${version}";
+    sha512 = "1ybf71jwr781c7q3xvjla2vhnarba6dpp94ghzd31vhdc038g36f0ad2xp3hh8bk1hmlwzzqpcd5pl23w0f4b19kgms4cbk8rh6nvpr";
```

Comment:
Any reason why you chose `sha512`? Nixpkgs usually defaults to `sha256`.

It doesn't matter (I don't think); just curious.